### DISCUSSION TITLE
Automatic backup

### DISCUSSION THREAD

**Discussion by joaochris** [sws2apps/organized-app#74] [Q&A] [answered]
I would like to know if the automatic backup is stored in the same place as the manual one.

How to recover an automatic backup?

**Comment by rhahao** [comment, marked as answer]
Hi @joaochris, yes, they are saved in the same place. The restore action is the same. It’s under the Congregation card > Restore Backup.

**Reply by joaochris** [reply]
For me when restoring only the manual backup appears 🤔

I say this because the most recent date does not appear. The date I made the manual backup appears.

**Reply by rhahao** [reply]
mmh... Let me double check that. It could be that the last automatic backup did not succeed because of the error you posted yesterday.

**Reply by rhahao** [reply]
@joaochris: I’ve checked this, and I am not able to duplicate it. Can you please confirm if you still have the issue?

**Reply by joaochris** [reply]
@rhahao When I click on create backup, does the automatic backup date appear?

**Reply by rhahao** [reply]
Hi @joaochris, yes, it should display the date of the last backup, whether manual or automatic.

**Reply by rhahao** [reply]
I’ve checked logs and I can see that the latest successful backup you have is on 27 Feb. I would like to suggest if you can logout from online account and reconnect again. Hopefully that should solve it.

**Reply by rhahao** [reply]
@joaochris: thanks to one friend here, I finally found why your backup is always failing 😁. I will prepare a fix for this asap.

**Reply by joaochris** [reply]
I was about to comment that it wasn't working yet lol. Thanks :)

**Reply by rhahao** [reply]
@joaochris: I hope finally that this should be solved now. Can you please confirm? 😁

**Reply by joaochris** [reply]
Now yes! Thanks! One question, now the time when the backup was made does not appear anymore?

**Reply by rhahao** [reply]
Oh... really? Do you have a screenshot or is there an error in the console?

**Reply by joaochris** [reply]
There are no console errors.
![Screenshot_19](https://user-images.githubusercontent.com/76891864/224835551-fb8015d5-3bd2-43c8-a2f7-ac437d469ef2.png)

**Comment by rhahao** [comment]
Hi @joaochris, I hope that all is fine for now! Backup is now working again, and the format used to display the date was adjusted.

**Reply by joaochris** [reply]
Now all right! Thanks :)
